MultiTabbedPDFViewer/ViewModel/PdfViewerViewModel.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MultiTabbedPDFViewer
+{
+    public class PdfViewerViewModel : INotifyPropertyChanged
+    {
+        /// <summary>
+        /// Constructor of the view model class
+        /// </summary>
+        public PdfViewerViewModel()
+        {
+            //Accessing the PDF document that is added as embedded resource as stream.
+            _documentSource = typeof(App).GetTypeInfo().Assembly.GetManifestResourceStream("MultiTabbedPDFViewer.Assets.PDF_Succinctly.pdf");
+            _documentSource2 = typeof(App).GetTypeInfo().Assembly.GetManifestResourceStream("MultiTabbedPDFViewer.Assets.form_document.pdf");
+        }
+
+        private Stream? _documentSource;
+
+        /// <summary>
+        /// The PDF document stream that is loaded into the instance of the PDF viewer. 
+        /// </summary>
+        public Stream? PDFDocumentSource
```

Comment:
Instead of multiple properties of document source, use one property as array of document source